src/main/java/httpserver/request/RequestParser.java

```diff
@@ -19,6 +16,7 @@ public Request parse(InputStream httpRequest) throws IOException {
 
        setStartLine(startLine, requestBuilder);
        setHeaders(requestBuilder);
+       setBody(contentLength, requestBuilder);
        return requestBuilder.build();
```

Comment:
This reads a lot better and doesnt hide the body creation in the header section. Nice refactor.